articles/api/management/v2/create-m2m-app.md

```diff
@@ -25,15 +25,14 @@ To create and authorize a Machine-to-Machine Application for the Management API:
 The application created in the steps above has been granted __all__ the Management API <dfn data-key="scope">scopes</dfn>. This means that it can access all endpoints.
 
 ::: panel How can I find out which scopes/permissions are required?
-Each machine-to-machine application that accesses an API must be granted a set of <dfn data-key="scope">scopes</dfn>. Scopes are permissions that should be granted by the owner. Each [Auth0 Management API v2](/api/management/v2) endpoint requires specific scopes. To see the required scopes/permissions for each endpoint, go to the [Management API Explorer](/api/management/v2#!) and find the endpoint you want to call. Each endpoint has a section called **Scopes** listing all the scopes that the endpoint requires. For example, the [Get all clients](/api/management/v2#!/Clients/get_clients) endpoint requires the scopes `read:clients` and `read:client_keys`.
+Each machine-to-machine application that accesses an API must be granted a set of <dfn data-key="scope">scopes</dfn>. Scopes are permissions that should be granted by the owner. Each [Auth0 Management API v2](/api/management/v2) endpoint requires specific scopes. To see the required scopes/permissions for each endpoint, go to the [Management API Explorer](/api/management/v2#!) and find the endpoint you want to call. Each endpoint has a section called **Scopes** listing all the scopes that the endpoint accepts.
 :::
 
-## Example: Get All Clients Endpoint
+## Example: Get All Connections Endpoint
 
-The [Get all clients](/api/management/v2#!/Clients/get_clients) endpoint requires the scopes `read:clients` and `read:client_keys`, while the [Create an application](/api/management/v2#!/Clients/post_clients) endpoint requires the scope `create:clients`. From that we can deduce that if we need to read _and_ create applications, then our token should include three scopes: `read:clients`, `read:client_keys` and `create:clients`.
-
-If you have multiple applications that should access the Management API, and you need different sets of scopes per app, we recommend creating a new machine-to-machine application for each one. For example, if one application is to read and create users (`create:users`, `read:users`) and another to read and create applications (`create:clients`, `read:clients`) create two applications (one for user scopes, one for applications) instead of one.
+The [Get all connections](/api/management/v2#!/Connections/get_connections) endpoint accepts the `read:connections` scope, whilst the [Create a connection](/api/management/v2#!/Connections/post_connections) endpoint accepts the `write:connections` scope. Using that we now know that our machine-to-machine token should only need the `read:connections` scope in order to access data from that endpoint.
```

Comment:
Hi. Recent edit across Docs is to update endpoint names [Get All Connections] [Create a Connection] like you can find in Authentication API Explorer here: https://auth0.com/docs/api/authentication#get-code-or-link. 
Also, 'while' instead of 'whilst' and you don't need the comma there.
This sentence is a little awkward: "Using that we now know..." so maybe something like, "Our machine-to-machine token should only need the...."

Reply:
Thank you @avanscoy, I've made some changes. 

However I'm a little bit confused by...
> Recent edit across Docs is to update endpoint names [Get All Connections] [Create a Connection] like you can find in Authentication API Explorer here: https://auth0.com/docs/api/authentication#get-code-or-link.

I'm unsure what you're referring to on the https://auth0.com/docs/api/authentication#get-code-or-link and what is your proposed change to this document regarding this. Could you re-phase what your suggestion is please, sorry I might be missing something obvious :)

Reply:
Thanks for your comment in slack that has helped, I've fixed the capitalisation on the endpoints.